### PR TITLE
Update docs for `doslearn`

### PR DESCRIPTION
**Pull request by jwa7**

### CODE REVIEW

**Comment by HowWeiBin** [comment]
Finished all of the actionables we decided on a few weeks back, please review and merge whenever you are free to do so

File: example/doslearn-aims-tutorial/part-2-ml/dft-options.yaml
Comment (jwa7) [inline example/doslearn-aims-tutorial/part-2-ml/dft-options.yaml:1]: This file doesn't need to be here, it's copied from part-1 by the user when doing the tutorial

File: example/doslearn-aims-tutorial/part-2-ml/part-2-ml.ipynb
Comment (jwa7) [inline example/doslearn-aims-tutorial/part-2-ml/part-2-ml.ipynb:1]: Please could you remove the outputs from these cells?

File: example/doslearn-aims-tutorial/part-1-dft/.ipynb_checkpoints/part-1-dft-checkpoint.ipynb
Comment (jwa7) [inline example/doslearn-aims-tutorial/part-1-dft/.ipynb_checkpoints/part-1-dft-checkpoint.ipynb:1]: Please remove this too

**Review by jwa7 — COMMENTED** [review]
Looks great, thanks! Just a couple of comments :)

File: example/doslearn-aims-tutorial/part-2-ml/dft-options.yaml
Comment (HowWeiBin) [reply example/doslearn-aims-tutorial/part-2-ml/dft-options.yaml:1]: done

File: example/doslearn-aims-tutorial/part-2-ml/part-2-ml.ipynb
Comment (HowWeiBin) [reply example/doslearn-aims-tutorial/part-2-ml/part-2-ml.ipynb:1]: done

File: example/doslearn-aims-tutorial/part-1-dft/.ipynb_checkpoints/part-1-dft-checkpoint.ipynb
Comment (HowWeiBin) [reply example/doslearn-aims-tutorial/part-1-dft/.ipynb_checkpoints/part-1-dft-checkpoint.ipynb:1]: done

**Comment by HowWeiBin** [comment]
Fixed all comments! Ready to merge anytime

**Comment by jwa7** [comment]
Thanks! Almost there - I've just rebased on top of the main branch. Please could you pull the changes and run `tox -e lint` and `tox -e format`, and make sure the tutorial runs ok with the changes? :)

**Pull request merged**